adapters/facebook.go

```diff
@@ -125,6 +125,24 @@ func (a *FacebookAdapter) MakeOpenRtbBidRequest(req *pbs.PBSRequest, bidder *pbs
 		}
 		fbReq.Imp[0].TagID = placementId
 
+		// if instl = 1 sent in, pass size (0,0) to facebook
+		if fbReq.Imp[0].Instl == 1 && fbReq.Imp[0].Banner != nil {
+			fbReq.Imp[0].Banner.W = 0
+			fbReq.Imp[0].Banner.H = 0
+		}
+		// if instl = 0 and type is banner, do not send non supported size
+		if fbReq.Imp[0].Instl == 0 && fbReq.Imp[0].Banner != nil {
+			supportedHeight := map[uint64]bool{
```

Comment:
do you know offhand if the Go compiler is smart enough to extract this and allocate just one map? You could probably test it by checking if `supportedHeight` has the same memory address on subsequent calls to this function.

If not, you can eek out some efficiency by declaring this map outside of the function, so that it only gets allocated once.